src/components/Notifications/form/event.ts

```diff
@@ -457,9 +454,9 @@ export const getEventSchema = (
       items: {
         "ui:ObjectFieldTemplate": RegexSelectorRow,
         regexSelect: {
-          "ui:widget": AntdSelect,
           "ui:data-cy": "regex-select",
```

Comment:
The regex selector doesn't exactly fit quite right within the modal

<img width="675" alt="Screen Shot 2022-12-13 at 12 49 36 PM" src="https://user-images.githubusercontent.com/47064971/207407564-b3a62776-b5e7-4323-98c3-2250dca801c1.png">

Reply:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/4605522/207409789-4df1c04a-996a-4414-a4f4-47cf1ad483f3.png">
Fixed Little bit of word foo

Reply:
sorry i accidentally submitted this comment as a standalone 😔 added more comment below